matchbox_demo/src/box_game.rs

```diff
@@ -141,10 +153,10 @@ pub fn increase_frame_system(mut frame_count: ResMut<FrameCount>) {
 #[allow(dead_code)]
 pub fn move_cube_system(
     mut query: Query<(&mut Transform, &mut Velocity, &Player), With<Rollback>>,
-    inputs: Res<Vec<GameInput>>,
+    inputs: Res<Vec<(BoxInput, InputStatus)>>,
```

Comment:
This resource looks a bit magical. Perhaps wrap it in some new type with more convenient accessors?

Guess it's more of a comment for bevy_ggrs, though.

Reply:
It is what ggrs returns, but it could be wrapped inside some syntactic sugar. This would have to be done inside bevy_ggrs, though.